src/cryptography/x509.py

```diff
@@ -976,6 +982,11 @@ def __ne__(self, other):
         return not self == other
 
 
+class SubjectAlternativeName(GeneralNames):
```

Comment:
Yup! `GeneralNames` should be a new class that `SubjectAltNames` composes around and proxies the methods, not subclasses.